storage/storage.cpp

```diff
@@ -770,6 +771,11 @@ void Storage::RegisterCountryFiles(TIndex const & index, string const & director
   TLocalFilePtr localFile = GetLocalFile(index, version);
   if (localFile)
     return;
+  // @TODO(bykoianko) This place should be implemented faster.
+  // You use Find twice. Here and in GetCountryFile.
```

Comment:
BTW, this code is called only when user deletes an mwm, so this place is not a bottleneck in our app.